### DISCUSSION TITLE
How do I disable this empty tab being created?

### DISCUSSION THREAD

**Discussion by kuro68k** [piroor/treestyletab#3665] [Q&A]
I have Gmail as a pinned tab, and sometimes when I open links from it in a new tab I get this extra one opened as well:

https://preview.redd.it/how-do-i-stop-tree-style-tab-opening-this-random-tab-v0-l1dkq6qe2o0e1.png?width=2188&format=png&auto=webp&s=1660613b2b7cb6d6994b4581962607d548a09921

How can I stop this tab opening? I'd rather those tabs were children of the Gmail one, or if that isn't possible just independent tabs.

**Comment by irvinm** [comment]
Do you have this enabled?  

![image](https://github.com/user-attachments/assets/eca283c8-0258-484d-8926-77075581cf5a)

**Reply by kuro68k** [reply]
Ah, thanks, that seems to be it. The number of options is a bit overwhelming.
